packages/react/src/components/Button/SplitButton/SplitButton.styles.ts

```diff
@@ -27,92 +27,128 @@ export const getStyles = memoizeFunction((theme: ITheme, customStyles?: IButtonS
-            borderTopRightRadius: '0',
-            borderBottomRightRadius: '0',
-            border: 'none',
-            flexGrow: '1',
+        '.ms-Button--default': {
+          borderTopRightRadius: '0',
+          borderBottomRightRadius: '0',
+          borderRight: 'none',
+          flexGrow: '1',
+        },
+        '.ms-Button--primary': {
+          borderTopRightRadius: '0',
+          borderBottomRightRadius: '0',
+          border: 'none',
+          flexGrow: '1',
 
-            selectors: {
-              [HighContrastSelector]: {
-                color: 'WindowText',
-                backgroundColor: 'Window',
-                border: '1px solid WindowText',
-                borderRightWidth: '0',
-                ...getHighContrastNoAdjustStyle(),
-              },
-              ':hover': {
-                border: 'none',
-              },
-              ':active': {
-                border: 'none',
+          [HighContrastSelector]: {
+            color: 'WindowText',
+            backgroundColor: 'Window',
+            border: '1px solid WindowText',
+            borderRightWidth: '0',
+            ...getHighContrastNoAdjustStyle(),
+          },
+          ':hover': {
+            backgroundColor: 'Highlight',
+            borderColor: 'Highlight',
+            borderRightWidth: '0',
+            color: 'Window',
+          },
+          ':active': {
+            borderColor: 'Highlight',
+          },
+        },
+        '.ms-Button--default + .ms-Button': {
+          [HighContrastSelector]: {
+            border: '1px solid WindowText',
+            borderLeftWidth: '0',
+            ':hover': {
+              borderColor: 'Highlight',
+              color: 'Highlight',
+              '.ms-Button-menuIcon': {
+                color: 'Highlight',
               },
             },
           },
-          '.ms-Button--primary + .ms-Button': {
-            border: 'none',
-            selectors: {
-              [HighContrastSelector]: {
-                border: '1px solid WindowText',
-                borderLeftWidth: '0',
-              },
+        },
+        '.ms-Button--default + .ms-Button[aria-expanded="true"]': {
+          [HighContrastSelector]: {
+            borderColor: 'Highlight',
+            color: 'Highlight',
+            '.ms-Button-menuIcon': {
+              color: 'Highlight',
             },
           },
         },
-      },
-    ],
-    splitButtonContainerHovered: {
-      selectors: {
-        '.ms-Button--primary': {
-          selectors: {
-            [HighContrastSelector]: {
-              color: 'Window',
+        '.ms-Button--primary + .ms-Button': {
+          border: 'none',
+
+          [HighContrastSelector]: {
+            border: '1px solid WindowText',
+            borderLeftWidth: '0',
+            ':hover': {
+              borderLeftWidth: '0',
               backgroundColor: 'Highlight',
+              borderColor: 'Highlight',
+              color: 'Window',
```

Comment:
```suggestion
              color: 'HighlightText',
```
for pairing with `backgroundColor`